### DISCUSSION TITLE
Soy nuevo

### DISCUSSION THREAD

**Discussion by jhalcantarac** [community/community#19108] [Other Feature Feedback, Questions, & Ideas]
soy nuevo

**Comment by rflmedina** [comment]
Hola, bienvenido a github 👋

**Comment by rsaldanar** [comment]
Hola @jhalcantarac , requieres de alguna ayuda en Español?

**Comment by Josephglz** [comment]
Bienvenido a Github! Si necesitas ayuda, avisame.

**Comment by Miles-Arts** [comment]
Hola compañeros!

**Reply by rsaldanar** [reply]
Hola Miles-Arts

**Comment by lv323** [comment]
hola, soy nueva, como pongo la pag en español? es que todo esta en ingles

**Comment by PromiseYoung** [comment]
Bienvenido a github
